go/service/config.go

```diff
@@ -54,7 +54,11 @@ func (h ConfigHandler) GetExtendedStatus(_ context.Context, sessionID int) (res
 
 	res.Standalone = h.G().Env.GetStandalone()
 	res.LogDir = h.G().Env.GetLogDir()
-	res.Clients = h.G().ConnectionManager.ListAllLabeledConnections()
+
+	// Should work in standalone mode too
+	if h.G().ConnectionManager != nil {
+		res.Clients = h.G().ConnectionManager.ListAllLabeledConnections()
+	}
```

Comment:
I assume this is a different bug (status in standalone mode)?  No need to remove, just making sure.